### PR TITLE
Youtube PlayList Data Fetch

### PR DESCRIPTION
**Pull request by Abhishek-90**
Fetching Video data from ReactPlay youtube Playlist.

### CODE REVIEW

**Review by atapas — COMMENTED** [review]
@Abhishek-90 don't commit the yarn lock file

File: pages/index.js [hunk 1]
```diff
@@ -2,13 +2,26 @@ import Layout from "@/components/Layout";
...
-    { id: 0, src: "https://www.youtube.com/embed/1qfDkmtuWqg" },
-    { id: 1, src: "https://www.youtube.com/embed/b0eas9xxD-E" },
-    { id: 2, src: "https://www.youtube.com/embed/w0nd4ASTDdg" },
-  ];
+  const [reactPlayLive, setReactPlayLive] = useState([]);
+
+  useEffect(() => {
+    const getPlayListData = async () => {
+      const response = await fetch(
+        `https://www.googleapis.com/youtube/v3/playlistItems?key=${process.env.NEXT_PUBLIC_YOUTUBE_API_KEY}&part=id,snippet&maxResults=5&playlistId=${REACT_PLAYLIST_ID}`
```
Comment (atapas) [inline pages/index.js]: Let's show Top 3.

**Review by atapas — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 1 above)
Comment (Abhishek-90) [reply pages/index.js]: Made the changes. Now we are only fetching 3 latest videos from the playlist

**Comment by Abhishek-90** [comment]
> @Abhishek-90 don't commit the yarn lock file

Delete `yarn.lock` file

**Review by atapas — CHANGES_REQUESTED** [review]
Provide a border radius and why the images are not fitting in the iframe?

image

**Review by atapas — APPROVED** [review]
Looks Good for Merge

**Pull request merged**